Sources/Commands/SwiftPackageTool.swift

```diff
@@ -31,7 +31,20 @@ public struct SwiftPackageTool: ParsableCommand {
         abstract: "Perform operations on Swift packages",
         discussion: "SEE ALSO: swift build, swift run, swift test",
         version: SwiftVersion.currentVersion.completeDisplayString,
-        subcommands: [
+        subcommands: getSubCommands(),
+        helpNames: [.short, .long, .customLong("help", withSingleDash: true)])
+
+    @OptionGroup()
+    var swiftOptions: SwiftToolOptions
+
+    public init() {}
+}
+
+extension SwiftPackageTool {
+    // This way the feature flag can control if the new subcommads
+    // are visible
+    static func getSubCommands() -> [ParsableCommand.Type] {
```

Comment:
nit: Swift-style in general, get- is usually omitted, so just subCommands() or subCommandList()

Reply:
If so, then it should probably be a computed property...but I think that's already what `subcommands` is?  So perhaps `makeSubcommands()` (if this does need to be a function?).